### DISCUSSION TITLE
被三次元压垮

### DISCUSSION THREAD

**Discussion by XiaoDao7755** [share121/inter-knot#1762] [其他]
![IMG20240812203859](https://github.com/user-attachments/assets/881d9a46-4919-427c-a02f-f0e7a4b67119)
rt，打工是不可能打工的，这辈子不可能打工的，只能上上绳网接接委托才能维持生活这样子

**Comment by lllliufeng** [comment]
接·格瓦拉

**Comment by ghost** [comment]
你这饱和过高了吧, 感觉颜色太艳了

**Reply by XiaoDao7755** [reply]
可能是手机相机的问题，实际还是挺柔的
